HW02_A_Graph-Fails.Rmd

```diff
@@ -46,7 +57,15 @@ ggplot(data = mpg) +
 
 ```
 
+**The attribute *color = "blue"* is in the function aes(), so it is treated as another aesthetic, or a variable, but it is supposed to be an attribute.**

```

Comment:
Looks great! One thing that might help with formatting a markdown document: if you want to highlight part of the code in normal text, you can enclose the argument you want to point out in backticks. For example, here, you could have put: `color = "blue"`. This can help to separate code from normal text in a paragraph of a markdown file.

Reply:
Thanks for your advice!